OndaEDFSchemas.jl/src/OndaEDFSchemas.jl

```diff
@@ -170,7 +175,7 @@ end
 A Legolas-generated record type that represents a single annotation imported
 from an EDF Annotation signal.  The `value` field contains the annotation value
 as a string.
-
 """
+EDFAnnotationV1
```

Comment:
With the newer versions of Legolas you can attach the docstring directly on the `@version` macro usage